webextensions/javascript-apis.json

```diff
@@ -3453,7 +3453,7 @@
                     "version_added": "48.0"
                   },
                   "opera": {
-                    "version_added": false
+                    "version_added": true
```

Comment:
Alright. While on it, I noticed that the compatibility information for `matchAboutBlank` are inconsistent. `InjectDetails` and `tabs.removeCSS` state that `firefox: false, firefox_android: "54"`, which seems suspiciously wrong, while `tabs.insertCSS` and `tabs.executeScript` states that it's not supported on any Firefox. So if either of these are right, it's probably the latter, or it is meanwhile supported by Firefox but somebody only updated some of the `version_added` fields. Do you know by any chance?